### PR TITLE
[OSDOCS#9314]: 4.12.47 RN

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.12

Issue:
https://issues.redhat.com/browse/OSDOCS-9314

Link to docs preview:
https://70404--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes#ocp-4-12-47

QE review:
N/A

Additional information:
URL's will not work until 4.12.47 is live

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -4385,4 +4385,29 @@ $ oc adm release info 4.12.46 --pullspecs
...
+
+[source,terminal]
+----
+$ oc adm release info 4.12.47 --pullspecs
+----
+
+[id="ocp-4-12-47-bug-fixes"]
+==== Bug fixes
+
+* Previously, the `spec.storage.deviceClasses.thinPoolConfig.overprovisionRatio` value on a Logical Volume Manager Storage (LVMS) cluster custom resource could only be set to a minimum of 2. With this release, the `spec.storage.deviceClasses.thinPoolConfig.overprovisionRatio` value can now be set to as low as 1, which disables overprovisioning. (link:https://issues.redhat.com/browse/OCPBUGS-24480[*OCPBUGS-24480*])
```
Comment (skopacz1) [inline release_notes/ocp-4-12-release-notes.adoc:4407]: Tiny nit: I think if we're talking about values of specs, we want to put the actual values in code format:
```suggestion
* Previously, the `spec.storage.deviceClasses.thinPoolConfig.overprovisionRatio` value on a Logical Volume Manager Storage (LVMS) cluster custom resource could only be set to a minimum of `2`. With this release, the `spec.storage.deviceClasses.thinPoolConfig.overprovisionRatio` value can now be set to as low as `1`, which disables overprovisioning. (link:https://issues.redhat.com/browse/OCPBUGS-24480[*OCPBUGS-24480*])  
```

**Review by skopacz1 — COMMENTED** [review]
One tiny nit, otherwise LGTM!

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (obrown1205) [reply release_notes/ocp-4-12-release-notes.adoc]: good catch! done :)

**Review by skrthomas — COMMENTED** [review]
merge review lgtm. I think you just need QE approval to merge. I see you have QE is N/A but I was thinking that this adds a new bug fix, so shouldn't it be approved?

**Pull request merged**